department-of-veterans-affairs/caseflow
# Allow SCM user to view judge assign queue

Resolves #12423

### Description
Enable SCM user to view a judge's assign queue using a specific URL like any of these:
```
http://localhost:3000/queue/3/assign?scm=true&judge_css_id=BVAAABSHIRE
http://localhost:3000/queue/5/assign?scm=true&judge_css_id=BVAEBECKER
http://localhost:3000/queue/75/assign?scm=true&judge_css_id=CSS_ID41
```

Once feature flag `scm_view_judge_assign_queue` is enabled, only judges and SCM users will be able to access URLs like the above.

### Acceptance Criteria
- [ ] Code compiles correctly and tests pass

### Testing Plan
1. Log in as an SCM user (e.g., BVARDUNKLE)
2. Browse to any of the above URLs 
3. Check that these pages are exactly the same as if the judge was logged in and was viewing their case-assign page (e.g., `http://localhost:3000/queue/5/assign`). The only difference should be the "Assign N Cases for BVAEBECKER" heading, which was requested as part of the original ticket.

### User Facing Changes
